sentry/src/main/java/io/sentry/Span.java

```diff
@@ -17,7 +17,7 @@
    * A transaction this span is attached to. Marked as transient to be ignored during JSON
    * serialization.
    */
-  private final transient @NotNull ITransaction transaction;
+  private final transient @NotNull SentryTransaction transaction;
```

Comment:
why the change, just out of curiosity?

Reply:
Because a method to start span that takes parent span is not anymore there on the `ITransaction` as we don't want to pollute public API with it. `SentryTransaction` has it package private.